components/ILIAS/Forum/classes/class.ilForumLP.php

```diff
@@ -18,48 +18,58 @@
 
 declare(strict_types=1);
 
+use ILIAS\UI\Component\Input\Input;
+
 /**
  * Class ilForumLP
  * @author Michael Jansen <mjansen@databay.de>
  */
 class ilForumLP extends ilObjectLP
 {
-    public function appendModeConfiguration(int $mode, ilRadioOption $modeElement): void
+    /**
+     * @return Input[]
```

Comment:
@schmitz-ilias I will change the return type here, cause it is not a list, but `array<string, Input>`.

Reply:
Yeah that is much nicer, I'll do the same in Tracking. Thanks!